<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Post2dGridRegionSelectDialog</class>
 <widget class="QDialog" name="Post2dGridRegionSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Region Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="fullRadioButton">
     <property name="text">
      <string>Full Region</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="activeRadioButton">
     <property name="text">
      <string>Active Region</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="customRadioButton">
     <property name="text">
      <string>Custom Region</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="StructuredGridRegionSelectWidget" name="structuredGridRegionWidget" native="true">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>195</width>
         <height>132</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StructuredGridRegionSelectWidget</class>
   <extends>QWidget</extends>
   <header>guibase/structuredgridregionselectwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Post2dGridRegionSelectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Post2dGridRegionSelectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>structuredGridRegionWidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
